events/schema.py

```diff
@@ -414,7 +414,7 @@ class Input:
         translations = graphene.List(EventTranslationsInput)
         duration = graphene.Int()
         participants_per_invite = EventParticipantsPerInvite(required=True)
-        capacity_per_occurrence = graphene.Int(required=True)
+        capacity_per_occurrence = graphene.Int()
```

Comment:
I guess you could add a description here that it will be required if the ticket system is internal. Just for documentation purposes.

Reply:
Good idea, will do! 👍